subgraphs/enzyme-core/templates/addresses.ts

```diff
@@ -191,7 +192,9 @@ export let release4Addresses: Release4Addresses = {
     '{{releases.v4.cumulativeSlippageTolerancePolicyAddress}}',
   ),
   curveLiquidityAdapterAddress: Address.fromString('{{releases.v4.curveLiquidityAdapterAddress}}'),
-  entranceRateBurnFeeAddress: Address.fromString('{{releases.v4.entranceRateBurnFeeAddress}}'),
+  disallowedAdapterIncomingAssetsPolicyAddress: Address.fromString(
+    '{{releases.v4.disallowedAdapterIncomingAssetsPolicyAddress}}',
+  ),entranceRateBurnFeeAddress: Address.fromString('{{releases.v4.entranceRateBurnFeeAddress}}'),
```

Comment:
this formatting looks weird

Reply:
we could remove `/templates` from prettier ignore, and allow prettier to format it correctly

Reply:
Indeed. Fixed.